okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -476,6 +485,9 @@ public void run() {
         }
       } catch (IOException ioe) {
         abort(ioe);
+      } catch (RuntimeException e) {
+        abort(e);
+        throw e;
```

Comment:
This does a log+throw. You should generally only log or only throw (like is being done for the IOException catch).